Solutions/Marain.Workflow.Api.MessageProcessingHost/Marain/Workflows/Api/MessageProcessingHost/Activities/CreateWorkflowActivity.cs

```diff
@@ -19,24 +20,20 @@ namespace Marain.Workflows.Api.MessageProcessingHost.Activities
     /// </summary>
     public class CreateWorkflowActivity
     {
-        private readonly IConfiguration configuration;
-        private readonly IWorkflowEngineClient engineClient;
+        private readonly IMarainWorkflowEngine engineClient;
         private readonly IJsonSerializerSettingsProvider serializerSettingsProvider;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="CreateWorkflowActivity"/> class.
         /// </summary>
-        /// <param name="configuration">The current configuration.</param>
         /// <param name="serializerSettingsProvider">The serialization settings provider.</param>
         /// <param name="engineClient">The current client.</param>
         public CreateWorkflowActivity(
-            IConfiguration configuration,
             IJsonSerializerSettingsProvider serializerSettingsProvider,
-            IWorkflowEngineClient engineClient)
+            IMarainWorkflowEngine engineClient)
         {
             // TODO: Replace with custom config class.
```

Comment:
Since we've removed the `configuration` argument, can we now remove this TODO comment too?